app/helpers/acs-puma-cd-tooltip.js

```diff
@@ -0,0 +1,20 @@
+import { helper } from '@ember/component/helper';
+
+export function buildTooltip(data) {
+  const {
+    puma,
+    cd_short_title,
+    shared_puma,
+    shared_puma_cd,
+  } = data[0];
+
+  const acsTooltipPuma = `American Community Survey 2014-2018 5-Year Estimates for PUMA ${puma}, `;
```

Comment:
This is an interesting problem. We need to dynamically fill in these sentences, but are also trying to extract out as much tooltip text as possible and consolidate them in separate json or js files, so that text is separate from the code. That would ideally be the case if we eventually do introduce an internationalization addon.

Just food for thought: what if we take this one step farther, and extract out these dynamic sentences into the separate `.js` files? 

```js
  const applicationCopy = {
     tooltips: {
          acs: {
              puma: function(puma) {
                  return `American Community Survey 2014-2018 5-Year Estimates for PUMA ${puma}, `;
              },
              cdApproximation: function(cd_short_title) {
                   return: `which is an approximation of ${cd_short_title}`;
              }
              .... 
          }
      }
  }
```

Then the helper would just pull text from that file:
```js
   const concatenatedTooltip = shared_puma ? appText.tooltips.acs.cdApproximationShared(shared_puma)cd) : appText.tooltips.acs.cdApproximation(cd_approximation);
   return appText.tooltips.acs.puma(puma).concat(concatenatedTooltip)
```

Reply:
I like that idea! Especially since some of the text is similar across the tooltip helpers -- I can make that change